sw/device/silicon_creator/lib/manifest.h

```diff
@@ -81,6 +84,25 @@ typedef struct manifest {
    * Modulus of the signer's RSA public key.
    */
   sigverify_rsa_buffer_t modulus;
+  /**
+   * Offset of the start of the executable region of the image from the start
+   * of the manifest in bytes.
+   */
+  uint32_t code_start;
+  /**
+   * Offset of the end of the executable region (exclusive) of the image from
+   * the start of the manifest in bytes.
+   */
+  uint32_t code_end;
+  /**
+   * Offset of the first instruction to execute in the image from the start of
+   * the manifest in bytes.
+   */
+  uint32_t entry_point;
+  /**
+   * Trailing padding (due to image_timestamp and the size of the struct).
+   */
+  uint32_t padding;
```

Comment:
Should we pad this whole struct out to the next reasonable size for the manifest? (I think 1024 bytes?)

Reply:
I guess we can (that space is not used anyway due to the alignment requirement of `.vectors` ) but I'm not sure if we have to.